src/js/media-error.js

```diff
@@ -8,13 +8,13 @@ import assign from 'object.assign';
  *
  * @param {Number} code The media error code
  */
-let MediaError = function(code){
+let MediaError = function(code) {
```

Comment:
noting it here (but we shouldn't make this change as part of linting) but this potentially can cause a problem on IE8 since on line 22 we do `MediaError.defaultMessages` and IE8 is weird.